### PR TITLE
Running game tag

### PR DESCRIPTION
**Pull request by franziskas1981**
add a tag for a running game so I'm no longer confused

### CODE REVIEW

**Comment by PatrickRose** [comment]
This is still pretty noisy, but I'm just about to pull a `.prettier` config together which might help.

I _think_ I can rebase your branch to include that, so I'll do that first

**Review by PatrickRose — CHANGES_REQUESTED** [review]
Quick review done :) A couple of changes if that's alright?

Since I've mucked with your branch, you probably want to do

```
$ git checkout runningGameTag
$ git fetch
$ git reset --hard origin/runningGameTag
```

Or 

```
$ git checkout develop
$ git fetch
$ git branch -D runningGameTag
$ git checkout runningGameTag
```

File: src/pages/games/index.tsx [hunk 1]
```diff
@@ -43,6 +46,9 @@ function FetchedMegagame(props: Game) {
                     <Link href={`/games/${props._id}`}>
                         <a>{props.name}</a>
                     </Link>
+                    {getJSDateFromOmegaDate(props.date) < currentDate ? (
```
[the remark this answers is not in the document]
Comment (franziskas1981) [reply src/pages/games/index.tsx:49]: Yep... Will look at this tomorrow  and change it :)

[the remark this answers is not in the document]
Comment (franziskas1981) [reply src/pages/games/index.tsx]: image

**Comment by franziskas1981** [comment]
requested changes made.

**Comment by franziskas1981** [comment]
done... :P

**Review by PatrickRose — APPROVED** [review]

**Pull request merged**